concent_api/core/tests/test_integration_get_task_result.py

```diff
@@ -999,3 +999,116 @@ def test_concent_requests_task_result_from_provider_and_requestor_receives_task_
+        compute_task_def['deadline'] = 1512126000.999999
+        deserialized_task_to_compute = self._get_deserialized_task_to_compute(
+            compute_task_def=compute_task_def,
+            deadline="2017-12-01 11:00:00"
+        )
+        deserialized_report_computed_task = self._get_deserialized_report_computed_task(
+            task_to_compute=deserialized_task_to_compute,
+            timestamp="2017-12-01 11:00:01",
+        )
+        serialized_force_get_task_result = self._get_serialized_force_get_task_result(
+            report_computed_task=deserialized_report_computed_task,
+            timestamp="2017-12-01 11:00:01",
+        )
+        deserialized_force_get_task_result = load(
+            serialized_force_get_task_result,
+            CONCENT_PRIVATE_KEY,
+            self.REQUESTOR_PUBLIC_KEY,
+            check_time=False,
+        )
+
+        with freeze_time("2017-12-01 11:00:01"):
+            response_1 = self.client.post(
+                reverse('core:send'),
+                data=serialized_force_get_task_result,
+                content_type='application/octet-stream',
+            )
+
+        self.assertEqual(response_1.status_code, 200)
+        self._assert_stored_message_counter_increased(increased_by = 2)
+        self._test_subtask_state(
+            task_id='99',
+            subtask_id='8',
+            subtask_state=Subtask.SubtaskState.FORCING_RESULT_TRANSFER,
+            provider_key=self._get_encoded_provider_public_key(),
+            requestor_key=self._get_encoded_requestor_public_key(),
+            expected_nested_messages={'task_to_compute', 'report_computed_task'},
+            next_deadline=self._parse_iso_date_to_timestamp("2017-12-01 11:00:52"),
+        )
+        self._test_last_stored_messages(
+            expected_messages=[
+                message.tasks.TaskToCompute,
+                message.tasks.ReportComputedTask,
+            ],
+            task_id='99',
+            subtask_id='8',
+            timestamp="2017-12-01 11:00:01"
+        )
+        self._test_undelivered_pending_responses(
+            subtask_id='8',
+            client_public_key=self._get_encoded_provider_public_key(),
+            expected_pending_responses_receive=[
+                PendingResponse.ResponseType.ForceGetTaskResultUpload,
+            ]
+        )
+
+        # STEP 2: Provider receives force get task result and file transfer token inside ForceGetTaskResultUpload via Concent.
+        with freeze_time("2017-12-01 11:00:12"):
+            response_2 = self.client.post(
+                reverse('core:receive'),
+                data=self._create_provider_auth_message(),
+                content_type='application/octet-stream',
+            )
+
+        self.assertEqual(response_2.status_code, 200)
+        self._assert_stored_message_counter_not_increased()
+
+        message_from_concent = load(response_2.content, self.PROVIDER_PRIVATE_KEY, CONCENT_PUBLIC_KEY, check_time = False)
```

Comment:
it should be called `force_get_task_result_upload`

Reply:
Test deleted